Godeps

```diff
@@ -30,6 +30,8 @@ github.com/kardianos/osext c2c54e542fb797ad986b31721e1baedf214ca413
 github.com/kardianos/service 6d3a0ee7d3425d9d835debc51a0ca1ffa28f4893
 github.com/kballard/go-shellquote d8ec1a69a250a17bb0e419c386eac1f3711dc142
 github.com/klauspost/crc32 cb6bfca970f6908083f26f39a79009d608efd5cd
+github.com/lib/pq e182dc4027e2ded4b19396d638610f2653295f36
```

Comment:
the reason to use pq instead of other implementations is because on golang sql page https://github.com/golang/go/wiki/SQLDrivers is written that pq is included in and pass the compatibility test suite, the others not.
Anyway I can replace pq with pgx and perform simple tests